pkg/config/config.go

```diff
@@ -358,9 +358,10 @@ func GetYAML(data interface{}) (string, error) {
 
 // OpenPiperFile provides functionality to retrieve configuration via file or http
 func OpenPiperFile(name string, accessTokens map[string]string) (io.ReadCloser, error) {
-	if len(name) == 0 {
-		return nil, fmt.Errorf("no filename provided")
-	}
+	// TODO: Why does this break our unit tests?
```

Comment:
@OliverNocon After rebasing, we realized that your [merge](https://github.com/SAP/jenkins-library/pull/3443) broke our unit tests. Do you have any idea why this does not work anymore? The check you have introduced sounds reasonable...